notebooks/official/explainable_ai/sdk_custom_image_classification_online_explain.ipynb

```diff
@@ -1115,17 +1080,17 @@
       "source": [
         "### Explanation Specification\n",
         "\n",
-        "To get explanations when doing a prediction, you must enable the explanation capability and set corresponding settings when you upload your custom model to an Vertex `Model` resource. These settings are referred to as the explanation metadata, which consists of:\n",
+        "To get explanations when doing a prediction, you must enable the explanation capability and set corresponding settings when you upload your custom model to an Vertex AI model resource. These settings are referred to as the explanation metadata, which consists of:\n",
```

Comment:
change "an" to "a"

- when you upload your custom model to a Vertex AI model resource.